### DISCUSSION TITLE
how am i able to play 1v1.lol how do i access it

### DISCUSSION THREAD

**Discussion by HayoFN** [dragon731012/-WORKING-bookmarklets-and-games#12] [General]
how am I able to play 1v1.lol how do I access it? give link and instructions please

**Comment by holgrave777** [comment]
HayoFN you can acces it by going to this link https://github.com/dragon731012/-WORKING-bookmarklets-and-games/blob/main/games/1v1.lol-main.zip?raw=true then when you download it open the file app then click through untill you see index.htm then double click that and it will open and start loading

**Reply by holgrave777** [reply]
hope this helps

**Reply by HayoFN** [reply]
it did work thanks

**Reply by holgrave777** [reply]
your welcome

**Reply by sometgirldotonline** [reply]
> how the hell did you do that @holgrave777 what if i want a zip for [this website](https://tylerpalko.github.io/gamehub/10minutestilldawn/) or maybe [this website](https://tylerpalko.github.io/gamehub/adanceoffireandice/) like actually what wizardry is this

They just figure out everything it calls to - and download them and change urls to local

**Comment by holgrave777** [comment]
Wut its been a year since i told hayo and school bloked github now contact
my discord if u want to contact me chargyarados2 is my discord ur gonna
have to explain what u want cuz i posted that a yer ago dont contact me
befor 3:00 pm pst time

On Tue, Oct 3, 2023 at 10:33 PM Mazedotexe ***@***.***> wrote:

> how the hell did you do that @holgrave777 <https://github.com/holgrave777>
> what if i want a zip for this website
> <https://tylerpalko.github.io/gamehub/10minutestilldawn/> or maybe this
> website <https://tylerpalko.github.io/gamehub/adanceoffireandice/> like
> actually what wizardry is this
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dragon731012/-WORKING-bookmarklets-and-games/discussions/12#discussioncomment-7182703>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A5DXDOSOKYSPNSUPJ3WUDUDX5TYKFAVCNFSM6AAAAAAVMKKLQ2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TCOBSG4YDG>
> .
> You are receiving this because you were mentioned.Message ID:
> <dragon731012/-WORKING-bookmarklets-and-games/repo-discussions/12/comments/7182703
> @github.com>
>


-- 
          -Mysterious person

**Comment by holgrave777** [comment]
also idk my git password now im replyin throu email

On Wed, Oct 4, 2023 at 8:35 AM Mysterious person ***@***.***>
wrote:

> Wut its been a year since i told hayo and school bloked github now contact
> my discord if u want to contact me chargyarados2 is my discord ur gonna
> have to explain what u want cuz i posted that a yer ago dont contact me
> befor 3:00 pm pst time
>
> On Tue, Oct 3, 2023 at 10:33 PM Mazedotexe ***@***.***>
> wrote:
>
>> how the hell did you do that @holgrave777
>> <https://github.com/holgrave777> what if i want a zip for this website
>> <https://tylerpalko.github.io/gamehub/10minutestilldawn/> or maybe this
>> website <https://tylerpalko.github.io/gamehub/adanceoffireandice/> like
>> actually what wizardry is this
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/dragon731012/-WORKING-bookmarklets-and-games/discussions/12#discussioncomment-7182703>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/A5DXDOSOKYSPNSUPJ3WUDUDX5TYKFAVCNFSM6AAAAAAVMKKLQ2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TCOBSG4YDG>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> <dragon731012/-WORKING-bookmarklets-and-games/repo-discussions/12/comments/7182703
>> @github.com>
>>
>
>
> --
>           -Mysterious person
>
>

-- 
          -Mysterious person

**Comment by holgrave777** [comment]
or use goclone :)

On Tue, May 14, 2024 at 7:51 AM Nova ***@***.***> wrote:

> how the hell did you do that @holgrave777 <https://github.com/holgrave777>
> what if i want a zip for this website
> <https://tylerpalko.github.io/gamehub/10minutestilldawn/> or maybe this
> website <https://tylerpalko.github.io/gamehub/adanceoffireandice/> like
> actually what wizardry is this
>
> They just figure out everything it calls to - and download them and change
> urls to local
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dragon731012/-WORKING-bookmarklets-and-games/discussions/12#discussioncomment-9434969>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A5DXDOVK7YBMPQPJV5FOMULZCIQH5AVCNFSM6AAAAAAVMKKLQ2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TIMZUHE3DS>
> .
> You are receiving this because you were mentioned.Message ID:
> <dragon731012/-WORKING-bookmarklets-and-games/repo-discussions/12/comments/9434969
> @github.com>
>


-- 
          -Chargyarados

**Comment by holgrave777** [comment]
much easyer

On Tue, May 14, 2024 at 7:59 AM Charlie Dennison ***@***.***>
wrote:

> or use goclone :)
>
> On Tue, May 14, 2024 at 7:51 AM Nova ***@***.***> wrote:
>
>> how the hell did you do that @holgrave777
>> <https://github.com/holgrave777> what if i want a zip for this website
>> <https://tylerpalko.github.io/gamehub/10minutestilldawn/> or maybe this
>> website <https://tylerpalko.github.io/gamehub/adanceoffireandice/> like
>> actually what wizardry is this
>>
>> They just figure out everything it calls to - and download them and
>> change urls to local
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/dragon731012/-WORKING-bookmarklets-and-games/discussions/12#discussioncomment-9434969>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/A5DXDOVK7YBMPQPJV5FOMULZCIQH5AVCNFSM6AAAAAAVMKKLQ2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TIMZUHE3DS>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> <dragon731012/-WORKING-bookmarklets-and-games/repo-discussions/12/comments/9434969
>> @github.com>
>>
>
>
> --
>           -Chargyarados
>
>

-- 
          -Chargyarados
